slik_wrangler/dqa.py

```diff
@@ -93,12 +94,12 @@ def check(col1, col2):
     
     if len(duplicated_rows):
         log(
-            f"Dataframe contains duplicate rows that you should address. \n\ncolumns={list(dupulicated_rows.index)}\n", 
+            f"Dataframe contains duplicate rows that you should address. \n\ncolumns={list(duplicated_rows.index)}\n", 
```

Comment:
I remember I fixed this somewhere.

Reply:
Did you pull the latest changes before making a commit?

Reply:
Nope I didn't

Reply:
lol always pull changes first before making a commit.